src/bidders/prebid/index.js

```diff
@@ -120,35 +122,28 @@ export class Prebid extends BaseBidder {
 		let slotParams = {};
 
 		const slotAlias = context.get(`slots.${slotName}.bidderAlias`) || slotName;
-
-		if (window.pbjs && typeof window.pbjs.getBidResponsesForAdUnitCode === 'function') {
-			const bids = window.pbjs.getBidResponsesForAdUnitCode(slotAlias).bids || [];
-
-			if (bids.length) {
-				let bidParams = null;
-				const priorities = getPriorities();
-
-				bids.forEach((param) => {
-					if (param.status === 'rendered') {
-						return;
-					}
-
-					if (!bidParams) {
-						bidParams = param;
-					} else if (bidParams.cpm === param.cpm) {
-						if (priorities[bidParams.bidder] === priorities[param.bidder]) {
-							bidParams = bidParams.timeToRespond > param.timeToRespond ? param : bidParams;
-						} else {
-							bidParams = priorities[bidParams.bidder] < priorities[param.bidder] ? param : bidParams;
-						}
+		const bids = getBidByAdUnitCode(slotAlias, false);
```

Comment:
s/getBidByAdUnitCode/getBidsByAdUnitCode

Reply:
Where `rendered` flag is needed?

Reply:
Renamed.

Reply:
Right, I was using this in some try during development. It's not needed now.